rotkehlchen/chain/substrate/utils.py

```diff
@@ -35,6 +35,10 @@ def is_valid_substrate_address(
     External Address Format (SS58) documentation:
     https://github.com/paritytech/substrate/wiki/External-Address-Format-(SS58)
     """
+    if value.startswith('0x'):
```

Comment:
This is a bug with ss58_decode I suppose. Can you open a bug in their tracker and link it here? We should remove this check as it's not our job, but the library' s job to check this

Reply:
[Already did](https://github.com/polkascan/py-scale-codec/issues/27). Not a bug per se, but it does not work for us as intended anymore.